project2/index.html

```diff
@@ -5,20 +5,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
     <title>Watch National Treasure</title>
 </head>
 
 <body>
-    <div class="box box-1">Directed and produced by John Turteltaub! Rotten Tomatoes rates it 45%.</div>
-    <figure>
-        <img src="nick-cage-in-national-treasure-promo.jpg" alt="Nick Cage in National Treasure">
-        <figcaption>Nick Cage stars in National Treasure</figcaption>
-    </figure>
-    <div class="boxed">
-        Nick Cage stars in National Treasure.
+    <div class="card">
+        <h1>National Treasure</h1>
+        <h2>"action-packed" - Risa Benedyk</h2>
+        <h3>Rotten Tomatoes gives it 45%.</h3>
+        <h4>Directed and produced by John Turteltaub!</h4>
+        <h5>RATED PG-13</h5>
```

Comment:
Remember to avoid using ALL CAPS in markup. It's okay for PG-13, since that's how it should be written, but the RATED is a style choice and, therefore, should be controlled with CSS (`text-transform: uppercase;` does this).